app/views/sites/shared/_manage_display_sub_nav.html.haml

```diff
@@ -2,10 +2,12 @@
   .title Manage Display
 %ul.nav.nav-list
   %li{ site_nav_css_class_hash('displays') }= link_to 'Display Overview', edit_site_display_path(@site)
-  %li{ site_nav_css_class_hash('visual_designs') }= link_to t('sites.visual_designs.edit.visual_design',
+  - if @site.use_redesigned_results_page
```

Comment:
```suggestion

```

The `if` here is incorrect.  We need affiliate admins to be able to see and update their visual design settings all the time, not just after they've been opted in to using the redesign.